import_upstream.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export PYTHONPATH=/home/rosbuild/reprepro_updater/src
```

Comment:
The `refactor` branch also contains absolute paths. They should be replaced but not necessarily part of this PR.